src/olympia/blocklist/models.py

```diff
@@ -429,7 +446,7 @@ def process_input_guids(cls, guids, v_min, v_max, load_full_objects=True,
         Block.addon.average_daily_users,
         Block.min_version,
         Block.max_version,
-        Block.process_input_guids
+        Block.is_imported_from_kinto_regex
```

Comment:
Was this just a typo from before?

Reply:
yes!  I've no idea when it happened because `process_input_guids` has always been a `classmethod` rather than a property - find/replace fail or copypasta mistake.